README.md

```diff
@@ -16,42 +16,38 @@ API service and web client of the Registry of Open Community Challenges
 
 1. Create the file that contains the future environment variables
 
-       cp .env.sample .env
+       cp .env.example .env
 
-2. Export the variables defined in *.env* to environment variables
-
-       export $(grep -v '^#' .env | xargs -d '\n')
-
-3. Start the Data Node API service
+2. Start the Data Node API service
 
-       docker-compose up
+       docker-compose up --build
```

Comment:
We could add a note that if they want the service to run in the background, they could add the `-d` or `--detach` flag to the command.

Reply:
Done in next commit